COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
It might be easier to maintain a list of contributors like golang does: https://github.com/golang/go/blob/master/CONTRIBUTORS